### ISSUE TITLE
Tablet Pen Eraser Issue

### ISSUE THREAD

**Issue by soljak** [drawpile/Drawpile#341]
A fellow artist and I have just installed the newest version of DrawPile and have come to discover that the program will recognize the eraser side of the tablet pen once and then will not change back to the pen upon returning to the pen nib. This has occurred on both of our individual computers. It seems to be reset by shutting down DrawPile completely and then reopening it, however the same issue occurs again after the first use of the eraser.

In short, you get one normal use of the eraser before having to manually switch between the eraser and brush tools.

**Comment by ProcrastinatingDragon** [comment]
I'm having this exact problem too. Been trying to deal with it ever since I downloaded the new version of DrawPile. Its frustrating and very annoying. I'd love to see this fixed please.

**Comment by wamserma** [comment]
Can confirm with Drawpile 1.0.6 and Wacom Bamboo under Win 7. This feature request could be a workaround: https://github.com/callaa/Drawpile/issues/268

**Comment by callaa** [comment]
There is now a new version with experimental Windows Ink support, which should solve many tablet related problems people have had, so I'm now closing all open issues with the Tablet tag. See Issue #562 for details.
